gulpfile.js

```diff
@@ -11,16 +11,18 @@
   limitations under the License.
 */
 
+'use strict';
+
 /* eslint-env node */
 
-var gulp = require('gulp');
-var eslint = require('gulp-eslint');
-var source = require('vinyl-source-stream');
-var browserify = require('browserify');
-var babelify = require('babelify');
+const gulp = require('gulp');
+const eslint = require('gulp-eslint');
+const source = require('vinyl-source-stream');
+const browserify = require('browserify');
+const babelify = require('babelify');
 
-var build = function(entry) {
-  var bundler = browserify({
+const build = function(entry) {
+  let bundler = browserify({
```

Comment:
If we're going for `const` everywhere, then I think this one should be `const` too.